2.2.0/cstor-operator-2.2.0.yaml

```diff
@@ -0,0 +1,4147 @@
+      name: Backup/Schedule
+      type: string
+    - JSONPath: .spec.lastSnapName
+      description: Last successfully backup snapshot
+      name: LastSnap
+      type: string
+  group: cstor.openebs.io
+  names:
+    kind: CStorCompletedBackup
+    listKind: CStorCompletedBackupList
+    plural: cstorcompletedbackups
+    shortNames:
+      - ccompletedbackup
+    singular: cstorcompletedbackup
+  preserveUnknownFields: false
+  scope: Namespaced
+  subresources: {}
+  validation:
+    openAPIV3Schema:
+      description: CStorCompletedBackup describes a cstor completed-backup resource
+        created as custom resource
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: CStorCompletedBackupSpec is the spec for a CStorBackup resource
+          properties:
+            backupName:
+              description: BackupName is the name of backup or scheduled backup
+              type: string
+            lastSnapName:
+              description: LastSnapName is the name of last completed-backup's snapshot
+                name
+              type: string
+            secondLastSnapName:
+              description: SecondLastSnapName is the name of second last 'successfully'
+                completed-backup's snapshot
+              type: string
+            volumeName:
+              description: VolumeName is the name of volume for which this backup
+                is destined
+              type: string
+          type: object
+      required:
+        - spec
+      type: object
+  version: v1
+  versions:
+    - name: v1
+      served: true
+      storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+############################################################
+####                                                    ####
+####             CStorRestore CRD                       ####
+####                                                    ####
+############################################################
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorrestores.openebs.io
```

Comment:
here also same as above comment

Reply:
Yep. Will do that. Sorry I missed the other 2 CRDs.